modxcms/revolution
# Improve file system error feedback

### What does it do?
Creates/updates error messaging to be more accurate and descriptive when a file/folder can not be moved or otherwise changed. Also, in second commit, aligns the code formatting to our current rule set.

### Why is it needed?
Current error feedback is for the most part unactionable because it's too ambiguous.

### How to test

1. Create at least one file and one folder that have read-only permission.
2. Verify the new messaging appears and its language is clear by attempting to: a) edit (file), rename*, or move a read-only file/directory; or b) create new files/folders in your new read-only directory.

* Note that, interestingly, in many cases renaming or moving a read-only file is possible when you have write permissions on the parent/target directory. This is just how unix works and MODX currently doesn't attempt to apply the fine-grained measures that'd be necessary to lock down files to that degree.

### Related issue(s)/PR(s)
Resolves #16074
